LinkedListImplementation/Interfaces/ILinkedList.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace LinkedListImplementation.Interfaces
 {
-    interface ILinkedList<T> : IOrdinaryIterator<T>, IDescendingIterator<T>
+    public interface ILinkedList<T>
```

Comment:
Add IEnumerable please

Reply:
Is it necessary?

Reply:
You can't iterate your collection with LINQ without it